src/metldata/builtin_transformations/count_content_values/model_transform.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2021 - 2024 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Model transformation logic for the 'count content values' transformation"""
+
+from typing import Any, Final
+
+from schemapack.spec.schemapack import (
+    SchemaPack,
+)
+
+from metldata.builtin_transformations.common.model_transform import (
+    add_properties,
+)
+from metldata.builtin_transformations.count_content_values.instruction import (
+    CountContentValueInstruction,
+)
+
+DEFAULT_PROPERTY_SCHEMA: Final[dict[str, Any]] = {
+    "type": "object",
+    "additionalProperties": True,
+}
+
+
+def add_count_content(
```

Comment:
function name could reflect what it's doing a bit better, e.g. add_count_property_names, as it's not adding all of the content.